src/types/types.d.ts

```diff
@@ -13,3 +13,37 @@ export type NftMetadataType = {
 };
 
 export type NftAttributeType = { trait_type: string; value: string | number };
+
+export type NftHistoryType = {
+  token?: string;
+  from: string;
+  to?: string;
+  event: string;
+  timestamp: string;
+  txId: string;
+  marketplace?: string;
+  value?: string;
+};
```

Comment:
Why do we have so many optionals? I believe token, from, to, value are always present. It losens the type checks without adding value.